invenio_records_ui/permissions.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
```

Comment:
Which permission factory do you propose as default for `RECORDS_UI_DEFAULT_PERMISSION_FACTORY`?